src/main/java/no/unit/nva/cristin/projects/FetchCristinProjects.java

```diff
@@ -38,12 +39,14 @@
     public static final String LANGUAGE_QUERY_PARAMETER = "language";
     public static final String TITLE_QUERY_PARAMETER = "title";
     private static final String DEFAULT_LANGUAGE_CODE = "nb";
+    private static Environment environment = new Environment();
     private final transient CristinApiClient cristinApiClient;
     private final transient PresentationConverter presentationConverter = new PresentationConverter();
 
     @SuppressWarnings("unused")
+    @JacocoGenerated
     public FetchCristinProjects() {
-        this(new CristinApiClient(), new Environment());
+        this(new CristinApiClient(environment), environment);
```

Comment:
Maybe read the environment variables here and pass them into the client constructor?

Reply:
Have to call `this` first in constructor otherwise compiler error. Therefore I have to send straight in to CristinApiClient

Reply:
Maybe you can create a helper method that creates the client?

Reply:
Will do in another PR, have to get this one through, its already big enough